package obd2Package;

message OBD2 {
	optional int64 time = 1;
	optional double speed = 2;
	optional int32 rpm = 3;
}
